src/Runner.Common/Logging.cs

```diff
@@ -101,7 +101,7 @@ private void NewPage()
             EndPage();
             _byteCount = 0;
             _dataFileName = Path.Combine(_pagesFolder, $"{_timelineId}_{_timelineRecordId}_{++_pageCount}.log");
-            _pageData = new FileStream(_dataFileName, FileMode.CreateNew);
+            _pageData = new FileStream(_dataFileName, FileMode.CreateNew, FileAccess.Write, FileShare.ReadWrite);
```

Comment:
maybe `FileAccess.ReadWrite, FileShare.Read` to only allow other processes to read?

Reply:
From [docs](https://docs.microsoft.com/en-us/dotnet/api/system.io.filestream.-ctor?view=net-5.0#System_IO_FileStream__ctor_System_String_System_IO_FileMode_) `FileShare.Read is the default for those FileStream constructors without a FileShare parameter.`

I'm not clear why FileShare.ReadWrite would be needed, but others have indicated it helped solve the problem.  My only theory is that we set the share level to ReadWrite when opening additional file handles for uploading the file?

You are right though that the access should probably be ReadWrite.  The docs say the default when not specified for this FileMode is ReadWrite.